main/DatePicker/PickerCalendar.tsx

```diff
@@ -36,7 +37,7 @@ interface Props {
 }
 
 const PickerCalendar: FC<Props> = (props) => {
-  const { calendarWidth = 300 } = props;
+  const { calendarWidth = 300, locale = 'default' } = props;
```

Comment:
The default value of locale prop is 'default'. It is specified to PickerCalendar that is parent component of CalendarWeekDays.
@JeffGuKang